Tools/WinMLRunner/Main.cpp

```diff
@@ -146,7 +147,8 @@ HRESULT EvaluateModel(
     bool isGarbageData,
     const CommandLineArgs& args,
     OutputHelper& output,
-    bool capturePerf
+    bool capturePerf,
+    uint32_t iterNum
```

Comment:
can we change "iterNum" to "iterationNum"?

Reply:
Sure.